README.md

```diff
@@ -102,12 +121,29 @@ This component will look for atomic indicators of relationships. Atomic indicato
  (av_similar_to, VirusTotal, signature_similarity) |  
  (yara_similar_to, YARA, complex_AV_match) |  
 
-*Table 1: Definitions for primary relationships*
+*Table 2: Definitions for primary relationships*
 
 
 
 ###### ML Component
 
 This component will utilize ML algorithms to train models based on a labeled dataset and then assign every new unknown incoming artefact (depending on the type of artefact) to one of the trained malicious clusters/classes.
 
-#### Final Relationships Generator
+#### Final Relationships Score Generator
+
+###### Direct relationship score algorithm(WIP)
+Direct relationship score algorithm gives the similarity score between the query object and other object has direct relationship to query object. This algorithm considers relationship_type and weights in primary relationship table between two object.
```

Comment:
Add The at the beginning of the sentence. 
between the query object -> between the object queried
other object has direct relationship to query object -> the related objects. 
in -> from the
last object is plural